### DISCUSSION TITLE
The next release (6.2.62) and betas (6.2.61b) of Sympa

### DISCUSSION THREAD

**Discussion by ikedas** [sympa-community/sympa#1619] [Release works]
Hi @everyone.
  - Due date of the next stable (6.2.62) is proposed to be 20st Mar. 2021.
  - Maybe the first beta (6.2.61b.1) will come out in Feb.

Especially this release will finish fixes for CVE-2020-26880. See also [related thread](https://github.com/orgs/sympa-community/teams/everyone/discussions/17).

See also [the milestone 6.2.62](https://github.com/sympa-community/sympa/milestone/16).

**Comment by racke** [comment]
Good. It is the 20th March by the way. Any details about the fixes for CVE-2020-26880?

**Comment by ikedas** [comment]
I think fixes on the code have been done on sympa-community/sympa#946 (& sympa-community/sympa#1031).  Another fix we have to do is, I think, updating/adding documentation about safer configuration.

**Comment by xavierba** [comment]
March 20th seems a very distant date when a fix for such a bad security issue is available...
Shouldn't only that fix warrant a release asap ?

**Comment by ikedas** [comment]
This date was what I proposed and haven't been disagreed. See [the thread](https://github.com/orgs/sympa-community/teams/everyone/discussions/17).  Anyway I think we need slightly large amount of time to work for this fix --- In fact, updating documentation hasn't been done at all.

**Comment by ikedas** [comment]
### 6.2.61b.1 comes in 18 Feb

The first beta will be out.

- See [Closed issues in the next milestone](https://github.com/sympa-community/sympa/milestone/16?closed=1).

We have to work for documentation about critical changes, in addition to testing.  Thanks for help!

(Sorry but I'll be in trip for some days around this weekend.  That's why I decided to release beta in Thursday.)

**Comment by ikedas** [comment]
Done!

**Comment by ikedas** [comment]
@racke, How may we update #1074 ?

**Comment by ikedas** [comment]
### 6.2.62 is proposed to be 20th Mar. 2021

How are your process of updating the documentation going?

**Comment by racke** [comment]
Sorry I'm swamped at the moment, will try to take a look at the weekend.

**Comment by ikedas** [comment]
How is the works for documentation?

**Comment by ikedas** [comment]
I don't see any progress.

Release of 6.2.62 would be better to be postponed.

**Comment by racke** [comment]
Still lack of time for the documentation. I agree on postponing the release. Two weeks?

**Comment by ikedas** [comment]
I have no objection to two weeks.

(I can accept a postponement of up to a month.: I'm going on vacation for a week at the end of April. I think we'd be better to decide after a week, at 25 Mar., whether to postpone it again.)

**Comment by ikedas** [comment]
@racke, can you give a list of the documentation works you are going to do?

I have a plan to write an advisory about sympa-community/sympa#1091 .

**Comment by ikedas** [comment]
There seems to be no progress.  We'd be better ---
* to release 6.2.61b.2 by now,
* to postpone 6.2.62 one more week.

**Comment by ikedas** [comment]
Done!

**Comment by racke** [comment]
Sorry to ask, but which documentation is needed for the next release?

**Comment by ikedas** [comment]
I think you said we should document what will be changed on the next release.

**Comment by racke** [comment]
OK, will go sifting through the changes. Also going to prepare some documentation about avoid setuid.

**Comment by ikedas** [comment]
I'll author the advisory above in this week (sorry for delay).
Anyways I hope we'll release 6.2.62 around 20th Apr.

**Comment by racke** [comment]
OK I'm starting the release announcement draft.

**Comment by racke** [comment]
https://github.com/sympa-community/sympa/releases/tag/untagged-7ae71b2a9ec06e2bf202 (copy of 6.2.60).

I'll provide the information for the 6.2.62 release.

**Comment by ikedas** [comment]
I have no idea on how to do with Sympa-community/sympa#1074 . How should we do?

**Comment by racke** [comment]
I try to rebase it and and important information to the top. What should be the release date?

**Comment by ikedas** [comment]
Thanks.  I don’t know the date by now. Please  read my posts in the last month.

**Comment by ikedas** [comment]
Can we close sympa-community/sympa#1009 , and why? If we can’t, what should we do?

**Comment by ikedas** [comment]
Check sympa-community/sympa-community.github.io#80 .
For discussion please post to sympa-community/sympa#1091 .

**Comment by ikedas** [comment]
I decoded closing the comments on the advisory above at  24th Apr, 0 a.m. UTC.

**Comment by ikedas** [comment]
Sorry for delay.  If no objection, **6.2.62 will be released in the morning of 27th April (UTC)**.

**Comment by racke** [comment]
Sounds good to me :+1:

**Comment by ikedas** [comment]
Done!
Thank you for help!

**Comment by racke** [comment]
@ikedas ... thanks a lot for the release !!!
